static/app/utils/replays/hooks/useInitialTimeOffsetMs.spec.tsx

```diff
@@ -18,6 +18,8 @@ const replay = TestStubs.ReplayRecord();
 
 const NOON = '2023-04-14T12:00:00';
 const FIVE_PAST_NOON = '2023-04-14T12:05:00';
+const FIVE_PAST_TIMESTAMP_S = String(new Date(FIVE_PAST_NOON).getTime() / 1000);
+const FIVE_PAST_TIMESTAMP_MS = String(new Date(FIVE_PAST_NOON).getTime());
```

Comment:
Maybe keep one of these as int to test that base case.

Reply:
query param parsing always returns string, so TS is guarding against this one